Cargo.toml

```diff
@@ -18,7 +18,7 @@ members = ["proto"]
 [features]
 default = []
 # Enable failpoints
-failpoint = ["fail"]
+failpoints = ["fail", "fail/failpoints"]
```

Comment:
I think `["fail/failpoints"]` is enough.

Reply:
Indeed, I tried and that works. I just learned something new here, thanks for the suggestion!